Zend/zend_execute.c

```diff
@@ -1622,7 +1622,7 @@ static zend_always_inline zval *zend_fetch_dimension_address_inner(HashTable *ht
 		switch (Z_TYPE_P(dim)) {
 			case IS_UNDEF:
 				zval_undefined_cv(EG(current_execute_data)->opline->op2.var, EG(current_execute_data));
-				/* break missing intentionally */				
+				/* break missing intentionally */
```

Comment:
Please, don't mix white space changes with semantic patches. commit this separately.